common/test/acceptance/pages/lms/teams.py

```diff
@@ -332,7 +337,8 @@ def __init__(self, browser, course_id, topic):
 
     def is_browser_on_page(self):
         """Check if we're on the create team page for a particular topic."""
-        return self.q(css='.team-edit-fields').present
+        return self.q(css='.team-edit-fields').present \
```

Comment:
Nit: I believe our policy is not to use ``\`` for wrapping long lines. See the PEP8 recommendations:

https://www.python.org/dev/peps/pep-0008/#maximum-line-length